jekyll/_cci2/dynamic-config.md

```diff
@@ -6,48 +6,48 @@ version:
 - Cloud
 ---
 
-You may find that instead of manually creating each and every individual CircleCI configuration per project, you would prefer to generate these configurations dynamically, depending on specific [pipeline]({{ site.baseurl }}/2.0/concepts/#pipelines)
-parameters or file-paths.
+Instead of manually creating an individual CircleCI configuration per project, you might prefer in some cases to generate these configurations dynamically, depending on specific [pipeline parameters]({{ site.baseurl }}/2.0/pipeline-variables/) or file paths.
```

Comment:
@sulami To make sure I'm following - is there a specific reason for using "values" instead of "parameters"? For instance I noticed that this [Discuss post](https://discuss.circleci.com/t/intro-to-dynamic-config-via-setup-workflows/39868) refers to pipeline parameters. It does look like either pipeline parameters or values could be used... would "pipeline variables" be a better option here?

Reply:
Pipeline parameters are declared and set by the user, pipeline values are implicitly declared and set by us. Technically there's no term encompassing both, though they can be used interchangeably everywhere. I'm okay with calling the union of those two sets pipeline variables.